### ISSUE TITLE
Codegen seems broken!

### ISSUE THREAD

**Issue by archmoj** [plotly/plotly.py#4870]
After PRs merged today it appears the codegen is not producing the correct outputs.

In the morning I was able to generate correct plotly.py code after https://github.com/plotly/plotly.py/pull/4860/commits/5497d178b8ea0c78e1709e9414faab4d0427e0f5

But now I can not locate this file: `packages/javascript/jupyterlab-plotly/package.json` i.e. to control  the version of plotly.js.

**Comment by marthacryan** [comment]
The new source of the plotly.js version is `packages/python/plotly/js/package.json` (I know this seems odd to be under `packages/python` but we're planning to reorganize directories soon to move `packages/python` to just `packages`). 

There are a few codegen commands in setup.py. Which one are you using?

**Comment by archmoj** [comment]
@marthacryan I am using these steps: 
https://github.com/plotly/plotly.py/blob/master/contributing.md#update-to-a-new-version-of-plotlyjs
```
$ cd packages/python/plotly
$ python setup.py updateplotlyjs
```

**Comment by archmoj** [comment]
As a result the traces and attributes we deleted in plotly.js v3.0.0-rc.0 are back e.g.
```
	plotly/graph_objs/_heatmapgl.py
	plotly/graph_objs/_pointcloud.py
	plotly/graph_objs/heatmapgl/
	plotly/graph_objs/layout/template/data/_heatmapgl.py
	plotly/graph_objs/layout/template/data/_pointcloud.py
	plotly/graph_objs/pointcloud/
	plotly/validators/_heatmapgl.py
	plotly/validators/_pointcloud.py
	plotly/validators/heatmapgl/
	plotly/validators/layout/template/data/_heatmapgl.py
	plotly/validators/layout/template/data/_pointcloud.py
	plotly/validators/pointcloud/
```

**Comment by archmoj** [comment]
Resolved by #4871.
